src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.cs

```diff
@@ -212,6 +217,45 @@ public static void HasColorAt(FileInfo screenshot, float x, float y, Color expec
 			}
 		}
 
+		public static void HasPixels(FileInfo screenshot, params ExpectedPixels[] expectations)
+		{
+			using (var bitmap = new Bitmap(screenshot.FullName))
+			{
+				foreach (var expectation in expectations)
+				{
+					var x = expectation.X;
+					var y = expectation.Y;
+
+					Assert.GreaterOrEqual(bitmap.Width, x);
+					Assert.GreaterOrEqual(bitmap.Height, y);
+
+					for (var py = 0; py < expectation.Values.GetLength(0); py++)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4919453)